docs/quickstart-and-usage.md

```diff
@@ -478,7 +478,7 @@ Stream UDS Policy deny logs and UDS Operator error logs:
 
 ## Scan
 
-{{% alert-note %}}> Scan is an ALPHA feature.
```

Comment:
We still refer to scan as alpha in the command descriptions:

https://github.com/defenseunicorns/uds-cli/blob/db96aa3f757071e769ca54d35fff69dfaa92a576/src/cmd/vendored.go#L84-L85

Reply:
Yeah it says it in the docs below that line as well
![Screenshot 2024-09-20 at 1 47 51 PM](https://github.com/user-attachments/assets/99967c98-6f92-4fad-8565-5e332ee81d94)

Reply:
^ that's fine, the `> Scan is an ALPHA feature` looked odd in the docs in that format so I removed it